demo/kitchen-sink/doclist.js

```diff
@@ -66,8 +66,32 @@ function makeHuge(txt) {
 }
 
 var docs = {
-    "docs/javascript.js": {order: 1, name: "JavaScript"},
-
+    "docs/javascript.js": "JavaScript",
+    "docs/AsciiDoc.asciidoc": "AsciiDoc",
+    "docs/clojure.clj": "Clojure",
+    "docs/coffeescript.coffee": "CoffeeScript",
+    "docs/coldfusion.cfm": "ColdFusion",
+    "docs/cpp.cpp": "C/C++",
+    "docs/csharp.cs": "C#",
+    "docs/css.css": "CSS",
+    "docs/curly.curly": "Curly",
+    "docs/dart.dart": "Dart",
+    "docs/diff.diff": "Diff",
+    "docs/dot.dot": "Dot",
+    "docs/ezhil.n": "Ezhil",
```

Comment:
Docs are added automatically based on modelist, so this file doesn't need to be changed